pkg/extract/extract.go

```diff
@@ -36,21 +36,22 @@ func uncompress(tarball, targetDir string, fileFilter func(string) bool) ([]stri
 	}
 	defer file.Close()
 
-	if strings.HasSuffix(tarball, ".tar.xz") || strings.HasSuffix(tarball, ".crcbundle") {
+	switch {
+	case strings.HasSuffix(tarball, ".tar.xz") || strings.HasSuffix(tarball, ".crcbundle"):
```

Comment:
I think I'd write this as
```
case strings.HasSuffix(tarball, ".tar.xz"):
fallthrough
case strings.HasSuffix(tarball, ".crcbundle"):
```
or maybe
```
case strings.HasSuffix(tarball, ".tar.xz"), strings.HasSuffix(tarball, ".crcbundle"):
```

Reply:
Nice! I didn't know about the syntax with a comma. Thanks! I changed it.